docs/organization/dynamic-sampling/index.mdx

```diff
@@ -0,0 +1,219 @@
+
+</Note>
+
+In most of the cases we recommend automatic mode. It is more convenient as it automatically adjusts projects according to their traffic volume.
+
+### Automatic mode
+
+When set to "automatic" mode, you can configure the target sample rate for your organization. This value will specify how much of the incoming traffic should be stored across all your projects. Underneath, Dynamic Sampling determines how this sample rate is distributed across all projects in your organization. You can see in the project list a preview of the project-specific rates.
+
+Automatic mode boosts the sample rate of lower-volume projects while simoultaneously reducing the sample rate of higher-volume projects. Sample rates are adjusted continuously to account for changes in event volume over time. This process operates in a short sliding window which helps with seasonal traffic patterns like Weekends, holidays, and night times.
+
+Within the sample rates determined for each project, Dynamic Sampling applies additional sampling priorities to further prioritize the most valuable data. You can read more about the sampling priorities in the [Dynamic Sampling Priorities](#dynamic-sampling-priorities) section.
+
+The project preview is based on the last 24 hours or 30 days of traffic, depending on the preview setting. This will show the dynamically allocated rates for the project taking into account the last 24 hours or 30 days of traffic. As the rates of the projects continuously change, choose 24 hours to have a more accurate view on your current sample rates and 30 days to see the averaged sample rates in your billing cycle.
+
+![The dynamic sampling mode and target sample rate setting.](./img/project-sample-rate-preview.png)
+
+Switching back to automatic mode from manual mode will start the recalibration process immediatly and resets the sample rates of each project in the organization to the calculated values. There is no way to restore the manually configured project sample rates after the switch.
+
+### Manual mode
+
+In the "manual" mode, you can configure a fixed sample rate for each project in your organization. This value will specify how much of the incoming traffic should be stored for the respective project. The sample rate for each project can be set between 0% and 100% and it defaults to 100% and are not adjusted by Dynamic Sampling.
+
+When the sample rates of each project is edited, an estimated overall organization sample rate is calculated as a weighted average across all projects. This value is always an estimation and changes according to volume within projects.
+
+If you want to change the sample rates for all projects at once, you an edit the overall organition sample rate. This will in turn adjust the sample rates of all projects to achieve the estimate rate. The maximum value is 100% and other projects will be adjusted accordingly to achieve the estimate rate.
```

Comment:
```suggestion
If you want to change the sample rates for all projects at once, you an edit the overall organization sample rate. This will proportionally adjust the sample rates of all projects to achieve the estimate rate. The maximum value is 100% and other projects will be adjusted accordingly to achieve the estimate rate. You can make final changes to the updated sample rates before saving all projects at once.
```